service/dap/server_test.go

```diff
@@ -364,11 +364,11 @@ func TestAttachStopOnEntry(t *testing.T) {
 		// 13 >> disconnect, << disconnect
 		client.DisconnectRequestWithKillOption(true)
 
-		// Both of these scenarios are somehow possible.
-		// Even though the program has an infininte loop,
-		// it apears that a halt can cause it to terminate.
 		// Since we are in async mode while running, we might receive messages in either order.
 		msg := client.ExpectMessage(t)
+		if _, isOutput := msg.(*dap.OutputEvent); isOutput {
```

Comment:
What kind of message could that be?

Reply:
The `s.logToConsole("Detaching and terminating target process")`